pkg/virtctl/credentials/addkey/addkey_test.go

```diff
@@ -269,13 +204,8 @@ var _ = Describe("Credentials add-ssh-key", func() {
 	})
 
 	It("should add second key to secret", func() {
-		updateSecret(kubeClient, secretName, func(secret *corev1.Secret) {
-			secret.Data = map[string][]byte{
-				"old-key.pub": []byte("old-key"),
-			}
-		})
+		patchSecret(kubeClient, secretName, patch.WithReplace("/data", map[string][]byte{"old-key.pub": []byte("old-key")}))
```

Comment:
Generally, I don't like using patches, because of the overhead of JSON encoding and decoding, and the possibility of a typo in the path parameter. They make sense when changing objects in a real cluster, where there can be other concurrent modifications. But here in unit tests, it looks like unnecessary overhead.

It is just my opinion, you don't need to change it.